### PR TITLE
Add support for reading virtual temp sensors on Aquacomputer Octo

### PR DESCRIPTION
**Pull request by aleksamagicka**
Add support for reading sixteen virtual temperature sensors on the Aquacomputer Octo. Also, note virt temp sensor support in driver docstring.

---

Checklist:

- [x] Adhere to the [development process]
- [x] Conform to the [style guide]
- [x] Verify that the changes work as expected on real hardware
- [x] Add automated tests cases
- [x] Verify that all (other) automated tests (still) pass
- [x] Update the README and other applicable documentation pages
- [ ] Update the `liquidctl.8` Linux/Unix/Mac OS man page
- [x] Update or add applicable `docs/*guide.md` device guides
- [ ] Submit relevant data, scripts or dissectors to https://github.com/liquidctl/collected-device-data

New CLI flag?

- [ ] Adjust the completion scripts in `extra/completions/`

New device?

- [ ] Regenerate `extra/linux/71-liquidctl.rules` (instructions in the file header)
- [ ] Add entry to the README's supported device list with applicable notes (at least `e`) and `git` MRLV

New driver?

- [x] Document the protocol in `docs/developer/protocol/`

[development process]: https://github.com/liquidctl/liquidctl/blob/main/docs/developer/process.md
[style guide]: https://github.com/liquidctl/liquidctl/blob/main/docs/developer/style-guide.md

### CODE REVIEW

**Review by jonasmalacofilho — COMMENTED** [review]
LGTM, but can you please add a note similar to https://github.com/liquidctl/liquidctl/commit/26338332082e10a860e7c81146bc3a02773383c2 to the docs?

**Comment by jonasmalacofilho** [comment]
Thanks!

File: docs/aquacomputer-octo-guide.md [hunk 1]
```diff
@@ -58,6 +62,8 @@ Aquacomputer Octo (experimental)
 └── Fan 8 current     0.00  A
 ```
 
+_Changed in 1.12.0: read virtual temperature sensors as well._
```
Comment (jonasmalacofilho) [inline docs/aquacomputer-octo-guide.md:65]: FYI, in the future: I just now **changed** and [documented](https://github.com/liquidctl/liquidctl/blob/main/docs/developer/process.md#newchanged-in-notes) how new notes will look:

```markdown
_Changed in git: read virtual temperature sensors as well._<br>
```

Comment (aleksamagicka) [reply docs/aquacomputer-octo-guide.md:65]: Woops, sorry :( Missed that.

**Pull request merged**